dali/test/python/operator_2/test_python_function_cleanup.py

```diff
@@ -0,0 +1,27 @@
+import nvidia.dali.fn as fn
```

Comment:
Do you have a license for that?

Reply:
Would you care to write one? It's your code ;)
Jokes aside, sure.